app/src/androidTest/java/chat/AndroidYggmail/delta/ExampleInstrumentedTest.java

```diff
@@ -1,4 +1,4 @@
-package org.cyberta;
+package chat.AndroidYggmail.delta;
```

Comment:
shouldn't the name change here to lower case as well? it doesn't look good with camel case

Reply:
this package name is completely wrong :D